docs/root/intro/version_history.rst

```diff
@@ -17,6 +17,7 @@ Version history
 * datasource: added retry policy for remote async data source.
 * dns: the STRICT_DNS cluster now only resolves to 0 hosts if DNS resolution successfully returns 0 hosts.
 * dns: added support for :ref:`dns_failure_refresh_rate <envoy_api_field_config.common.dynamic_forward_proxy.v2alpha.DnsCacheConfig.dns_failure_refresh_rate>` for the :ref:`dns cache <envoy_api_msg_config.common.dynamic_forward_proxy.v2alpha.DnsCacheConfig>` to set the DNS refresh rate during failures.
+* dynamic forward proxy: added SNI based :ref:`dynamic forward proxy <arch_overview_http_dynamic_forward_proxy>` support.
```

Comment:
Is it not also needed to add about SNI based dynamic forward proxy to `dynamic_forward_proxy_filter.rst`?

Reply:
+1 API LGTM modulo docs.

Reply:
added